spec/factories/case/offender_sar_complaints.rb

```diff
@@ -0,0 +1,143 @@
+FactoryBot.define do
+
+  factory :offender_sar_complaint, class: Case::SAR::OffenderComplaint do
```

Comment:
This got me thinking - I wonder if it would be nicer to have `Case::SAR::Offender::Complaint` rather than the `OffenderComplaint` class. 

What got me thinking this way is that we have a variable `offender_sar_complaint` and the class `OffenderComplaint` and it seems like there is a bit of a word order jumble there. Not sure if this helps but I thought I'd ask now as we are just starting 👍

Reply:
Great suggestion 👍